exe/ruby-lsp

```diff
@@ -36,6 +36,10 @@ parser = OptionParser.new do |opts|
     options[:experimental] = true
   end
 
+  opts.on("--doctor", "Run troubleshooting steps") do
```

Comment:
I didn't add a `-d` short prefix since it's not something that would be run often, and also to prevent unwanted clashes with other options we may want to add in future.